src/main/java/com/bgsoftware/superiorskyblock/island/SIsland.java

```diff
@@ -4852,16 +4852,14 @@ public int compareTo(Island other) {
         if (other == null)
             return -1;
 
-        SortingType sortingType = plugin.getSettings().getIslandTopOrder();
+        SortingType sortingType = SortingType.getByName(plugin.getSettings().getIslandTopOrder());
```

Comment:
This can be cached, and updated when settings is updated.
See `onSettingsUpdate`

Reply:
Because i just can add cache in SIsland but you would cache to defaultSorterType, so the best things is to save it on SorterTypes, but where i've to updates theses values ?